lib/actions/select.js

```diff
@@ -16,7 +16,16 @@ const ROWS_BATCH_SIZE = 1000;
  */
 function processAction(msg, cfg) {
   const self = this;
-  const { conString } = cfg;
+  const {
+    protocol,
+    host,
+    port,
+    databaseName,
+    user,
+    password,
+    configurationProperties,
+  } = cfg;
+  const conString = `${protocol}://${user}:${password}@${host}${port ? `:${port}` : ''}/${databaseName}${configurationProperties ? `?${configurationProperties}` : ''}`;
```

Comment:
I'd suggest to use native support of building configuration object instead of concatenation set of strings into the connection URI. It looks more clear.
https://node-postgres.com/features/connecting
```
const pool = new Pool({
  user: 'dbuser',
  host: 'database.server.com',
  database: 'mydb',
  password: 'secretpassword',
  port: 3211,
})
```
or
```
const client = new Client({
  user: 'dbuser',
  host: 'database.server.com',
  database: 'mydb',
  password: 'secretpassword',
  port: 3211,
})
```

Reply:
One more concern:
Usage of connection string have limitation of valid characters for `password`, for example, because they should be escaped.
That's why usage of connection object is preferable way.